modules/ossm-deploying-bookinfo-application-istio-ambient-mode.adoc

```diff
@@ -0,0 +1,120 @@
+
+.Prerequisites
+
+* You have deployed a cluster on {ocp-product-title} 4.15 or later.
+* You are logged in to the {ocp-product-title} either through the web console as a user with the `cluster-admin` role, or with the `oc login` command, depending on the installation method.
+* You have installed the {SMProductName} Operator, created the {istio} resource, and the Operator has deployed {istio}.
+* You have created an `IstioCNI` resource, and the Operator has deployed the necessary `IstioCNI` pods.
+* You have created a `Ztunnel` resource, and the Operator has deployed the necessary `Ztunnel` pods.
+
+.Procedure
+
+. Create the `bookinfo` namespace by running the following command:
++
+[source,terminal]
+----
+$ oc create namespace bookinfo
+----
+
+. Add the `istio-discovery=enabled` label to the `bookinfo` namespace by running the following command:
++
+[source,terminal]
+----
+$ oc label namespace bookinfo istio-discovery=enabled
+----
+
+. Apply the `bookinfo` YAML file to deploy the `bookinfo` application by running the following command:
++
+[source,terminal]
+----
+$ oc apply -n bookinfo -f https://raw.githubusercontent.com/istio/istio/release-1.26/samples/bookinfo/platform/kube/bookinfo.yaml
+----
+
+. Apply the `bookinfo-versions` YAML file to deploy the `bookinfo` application by running the following command:
++
+[source,terminal]
+----
+$ oc apply -n bookinfo -f https://raw.githubusercontent.com/istio/istio/release-1.26/samples/bookinfo/platform/kube/bookinfo-versions.yaml
+----
+
+. Verify that the `bookinfo` pods are running by entering the following command:
++
+[source,terminal]
+----
+$ oc -n bookinfo get pods
+----
++
+.Example output
+[source,terminal]
+----
+NAME                             READY   STATUS    RESTARTS   AGE
+details-v1-54ffdd5947-8gk5h      1/1     Running   0          5m9s
+productpage-v1-d49bb79b4-cb9sl   1/1     Running   0          5m3s
+ratings-v1-856f65bcff-h6kkf      1/1     Running   0          5m7s
+reviews-v1-848b8749df-wl5br      1/1     Running   0          5m6s
+reviews-v2-5fdf9886c7-8xprg      1/1     Running   0          5m5s
+reviews-v3-bb6b8ddc7-bvcm5       1/1     Running   0          5m5s
+----
+
+. Verify that the `bookinfo` application is running by entering the following command:
++
+[source,terminal]
+----
+$ oc exec "$(oc get pod -l app=ratings -n bookinfo \
+  -o jsonpath='{.items[0].metadata.name}')" \
+  -c ratings -n bookinfo \
+  -- curl -sS productpage:9080/productpage | grep -o "<title>.*</title>"
+----
+
+. Label the `bookinfo` namespace with `istio.io/dataplane-mode=ambient` to add the application to the mesh in ambient mode by running the following command:
++
+[source,terminal]
+----
+$ oc label namespace bookinfo istio.io/dataplane-mode=ambient
+----
++
+You can add the `istio.io/dataplane-mode=ambient` label to individual pods to include specific workloads.
```

Comment:
```suggestion
Alternatively, rather than including all workloads in a namespace, you can add the `istio.io/dataplane-mode=ambient` label to individual pods to include specific workloads. See the additional resources below for more details on the labels used to add or exclude workloads in a mesh.
```
Just wanted to clarify this statement a bit more.